ror-tools/src/main/scala/tech/beshu/ror/tools/RorToolsApp.scala

```diff
@@ -146,15 +143,16 @@ trait RorTools {
   private class VerifyCommandHandler(implicit inOut: InOut) {
     def handle(command: Command.Verify): Result = {
       val esDirectory = esDirectoryFrom(command.customEsPath)
-      handleResult(EsPatchExecutor.create(esDirectory).verify())
+      handleResult(EsPatchExecutor.create(esDirectory).verify(), errorMessagePrefix = "")
```

Comment:
This modification may require adjusting other tests too, not the single one that was adjusted (because the verify command may be executed also as an initial step of other commands, and may return errors in those context too)